runtime/shared_common/CacheMap.cpp

```diff
@@ -1049,7 +971,7 @@ SH_CacheMap::initializeROMSegmentList(J9VMThread* currentThread)
 	Trc_SHR_CM_initializeROMSegmentList_Entry(currentThread);
 
 	cacheBase = (U_8*)_ccHead->getBaseAddress();
-	firstROMClassAddress = _ccHead->getFirstROMClassAddress(_runningNested);
+	firstROMClassAddress = _ccHead->getFirstROMClassAddress(false);
```

Comment:
Pls remove the isNested parameter of getFirstROMClassAddress().

Reply:
Fixed